src/datatypes/nbt.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
Should be in prismarine-nbt (will need a PR for prismarine-nbt)

Reply:
done